### PR TITLE
Comply with rubocop (rubocop.yml)

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Enable: `Rails/I18nLocaleTexts` and exclude the files which cause this to fail. We will fix these up later. This will help to stop code rot.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by avinhurry** [comment]
[Comply with rubocop (rubocop.yml)](https://trello.com/c/82I5EYX6/1016-comply-with-rubocop-rubocopyml)

File: .rubocop.yml [hunk 1]
```diff
@@ -30,14 +26,6 @@ Naming/PredicateName:
 Rails/I18nLocaleTexts:
   Enabled: false
```
Comment (avinhurry) [inline .rubocop.yml:27]: Shall we card this up?

File: .rubocop.yml [hunk 2]
```diff
@@ -14,10 +14,6 @@ inherit_from:
 
 AllCops:
   NewCops: enable
-  Exclude:
-    - "bin/*"
-    - "db/schema.rb"
-    - "node_modules/**/*"
```
Comment (darokel) [inline .rubocop.yml:20]: I think we **want** to exclude these?

Comment (avinhurry) [reply .rubocop.yml:20]: Why? 🤔

Comment (darokel) [reply .rubocop.yml:20]: We don't care about Rubocop checking node modules as it's just JS stuff, same for bin, there's some usually some shell scripts in there. Db schema maybe?

Comment (avinhurry) [reply .rubocop.yml]: cool 👍🏼 - dropped the commit.

File: .rubocop.yml (hunk 1 above)
Comment (avinhurry) [reply .rubocop.yml]: https://trello.com/c/5yIVAAYg/343-enable-rails-i18nlocaletexts-and-fix-errors

**Review by defong — APPROVED** [review]

**Pull request merged**